### ISSUE TITLE
Need a way to enable optin check again if previously skipped.

### ISSUE THREAD

**Issue by danieliser** [Freemius/wordpress-sdk#22]

**Comment by vovafeldman** [comment]
What would you expect to happen from the user-experience perspective once the flag is changed?

**Comment by danieliser** [comment]
The way I see it this isn't a user experience issue. As a developer there should be a programmatic way to allow the optin to appear again without deleting the fs accounts option.

My method would simply add an option to my own settings that will be checked if they skipped optin. When they uncheck and save I should be able to clear the fs account flag for my plugin.

Use case. I have users that email asking if they can get a discount. I tell them to optin. Currently (pre freemius integration) they can simply go into settings and check the box. It will enroll them and kick out discount.

With freemius now, once they skip no way to redirect them back to opting in if they can be convinced.

Other potential use cases would be similar to your methods of obtaining reviews. After x popup opens and not opted in, ask if they would like to opt in and get discount to make their popups even better.

Lots of ways this could be useful, not to mention internally for your change of account routines if I'm not mistaken.

**Comment by vovafeldman** [comment]
Based on your use-case, it's not like you want to show the opt-in screen again. You actually want to programmatically opt-in/out the user. Am I right?

**Comment by danieliser** [comment]
@vovafeldman - Not really no. In my case if they skip during plugin update and then a week later email support asking for a discount. The only code we offer is via optin. So if they skipped I need a way that I can say go here, click this and then opt in.

Programatically I just want to clear the skip flag and then redirect back to the optin page when the user clicks reset button.

**Comment by vovafeldman** [comment]
Try to add the following function to your local environment and use it:
```php
		/**
		 * Clear anonymous mode and redirect to opt-in screen.
		 * 
		 * @author Vova Feldman (@svovaf)
		 * @since  1.1.7
		 */
		function connect_again() {
			if ( ! $this->is_anonymous() ) {
				return;
			}

			$this->reset_anonymous_mode();

			if ( fs_redirect( $this->get_activation_url() ) ) {
				exit();
			}
		}
```

If works as expected I'll push to master.

**Comment by danieliser** [comment]
@vovafeldman Tested. Works good.

**Comment by vovafeldman** [comment]
Merged in 62ec44e8b7055ff6920af74f409f7ec6798b224f

**Comment by danieliser** [comment]
@vovafeldman Awesome, ty sir. I will likely be adding this functionality to v1.4.1 of Popup Maker as I had to lock it up last night :).
